packages/webapp/src/components/AddCropVariety/index.jsx

```diff
@@ -73,14 +75,41 @@ export default function PureAddCropVariety({
     truncateText(crop.crop_genus, 22) + ' ' + truncateText(crop.crop_specie, 22);
   const progress = 33;
 
+  const uniqueVarietyName = () => {
+    const formVarietyName = getValues('crop_variety_name');
```

Comment:
`crop_variety_name` could be replaced with `COMMON_NAME`, but I will approve after doing some more testing!

Reply:
Ah yes, it definitely could. I used the hook form registered name in the Add Crop version of this component as no naming variables had been set up in that file. I would slightly prefer to keep the parallelism, but either way!